lib/index.js

```diff
@@ -4,7 +4,20 @@ const spawn = require('cross-spawn').spawn
 const path = require('path')
 const fs = require('fs')
 const os = require('os')
-const rcFileLocation = path.join(process.cwd(), '.env-cmdrc')
+
+const rcFilenames = [
+  '.env-cmdrc',
+  '.env-cmdrc.json'
+]
+let rcFileLocation = null
+
+rcFilenames.forEach((filename) => {
+  const rcPath = path.join(process.cwd(), filename)
+
+  if (fs.existsSync(rcPath)) {
```

Comment:
I believe we should check for the existence of `rcFileLocation` here as well. That way we don't overwrite the previously set version.